core/src/main/java/org/elasticsearch/action/DocWriteResponse.java

```diff
@@ -314,8 +307,8 @@ public XContentBuilder innerToXContent(XContentBuilder builder, Params params) t
                 .field(_ID, id)
                 .field(_VERSION, version)
                 .field(RESULT, getResult().getLowercase());
-        if (getTookInNanos() >= 0) {
-            builder.field(TOOK, getTookInMillis()); // Building 'took' in milliseconds for readability
+        if (getTook().nanos() >= 0) {
+            builder.field(TOOK, getTook().millis());    // Building 'took' in milliseconds for readability
```

Comment:
So would you prefer me to use `builder.timeValueField` instead or keep this section in its current state? 
Personally, I think the current form is concise and captures what we're trying to output intuitively. 
But I can definitely change it to follow the Elasticsearch style.